spark/src/main/scala/geotrellis/spark/io/accumulo/AccumuloRasterCatalog.scala

```diff
@@ -45,8 +45,12 @@ class AccumuloRasterCatalog(
         implicitly[RasterRDDReader[K]].read(instance, metaData, keyBounds, index)(layerId, filterSet)
       }
     }
-  
-  def writer[K: SpatialComponent: RasterRDDWriter: JsonFormat: Ordering: ClassTag](keyIndexMethod: KeyIndexMethod[K], tileTable: String): Writer[LayerId, RasterRDD[K]] = {
+
+  def writer[K: SpatialComponent: RasterRDDWriter: JsonFormat: Ordering: ClassTag](
+    keyIndexMethod: KeyIndexMethod[K],
+    tileTable: String,
+    strategy: AccumuloWriteStrategy
```

Comment:
We should default a write strategy here to avoid surprise. I think HDFS should be default for now until we resolve the performance problems on the BatchWriter strategy.